README.md

```diff
@@ -71,7 +74,7 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+...controller method.
```

Comment:
It would actually be a *helper* method. The purpose of helper methods -- like `form_for` and `link_to` -- is to generate some HTML, or something that is important only inside a view.